source/hardware/controllers/hercules_djcontrol_inpulse_200.rst

```diff
@@ -67,10 +67,10 @@ Browser
 ===============================================================  ==========================================
 Control                                                          Function
 ===============================================================  ==========================================
-:hwlabel:`BROWSER` encoder (turn)                                Move up/down list.
-:hwlabel:`SHIFT` + :hwlabel:`BROWSER` encoder (turn)             Scroll up/down list.
-:hwlabel:`BROWSER` encoder (press)                               Switch focus between list and file view.
-:hwlabel:`SHIFT` + :hwlabel:`BROWSER` encoder (press)            Maximize/Minimize library view.
+:hwlabel:`BROWSER` turn                                          Move up/down list.
+:hwlabel:`SHIFT` + :hwlabel:`BROWSER` turn                       Scroll up/down list.
+:hwlabel:`BROWSER` press                                         Switch focus between list and file view.
+:hwlabel:`SHIFT` + :hwlabel:`BROWSER` press                      Maximize/Minimize library view.
 :hwlabel:`ASSISTANT` button                                      Toggle AutoDJ (be sure a playlist was created for AutoDJ before activating this function).
```

Comment:
Ah, okay, the lines in the left column are broken because of the long text in the  second column.

This hint is not required IMO, but maybe it's obsessive to insist on a compact table here. Up to you :shrug: 
```suggestion
:hwlabel:`ASSISTANT` button                                      Toggle AutoDJ
```

Reply:
I want to fix it too! I tried to shorten it because I've been puzzled too by the autodj not working with an empty playlist.